lang/main-enGB.json

```diff
@@ -335,7 +335,8 @@
         "toggleFilmstrip": "Show or hide video thumbnails",
         "toggleScreensharing": "Switch between camera and screen sharing",
         "toggleShortcuts": "Show or hide keyboard shortcuts",
-        "videoMute": "Start or stop your camera"
+        "videoMute": "Start or stop your camera",
+        "toggleTileView": "Toggle tile view"
     },
```

Comment:
Please remove the changes from main-enGB.json, the only edited file through PRs is main.json, the rest is done through https://translate.jitsi.org.

Reply:
Removed changes from main-enGB.json.